### ISSUE TITLE
About loading pre-trained weights

### ISSUE THREAD

**Issue by GKYoffice** [ultralytics/ultralytics#16756]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Can I pass in the pre-trained weights downloaded locally (I am sure it is feasible in yolov8), such as using model = YOLO("./yolo11x.pt") in python; or input model=./yolo11x.pt in the command line. I am sure I downloaded the pre-trained weight file normally, but when running the training, I found that yolov11n.pt will be downloaded from github every time

How to solve it, thank you!
![1](https://github.com/user-attachments/assets/769b7ecf-81c6-4436-b77e-f73c633817dc)
![2](https://github.com/user-attachments/assets/80507354-9436-42f1-a557-2d4317f778d0)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @GKYoffice, thank you for your interest in Ultralytics 🚀! We recommend checking out our [Docs](https://docs.ultralytics.com) for new users, where you can find many examples for using [Python](https://docs.ultralytics.com/usage/python/) and the [CLI](https://docs.ultralytics.com/usage/cli/), which may have the answers you’re looking for.

If this is a 🐛 Bug Report, please consider providing a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us debug it effectively.

If you have a custom training ❓ Question, please provide additional information, including dataset image examples and any training logs. Make sure to follow our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best! For real-time chat, head to [Discord](https://ultralytics.com/discord) 🎧, or if you prefer in-depth discussions, check out [Discourse](https://community.ultralytics.com). You can also connect with others on our [Subreddit](https://reddit.com/r/ultralytics).

## Upgrade

Upgrade to the latest `ultralytics` package with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to ensure your issue hasn't been resolved in the latest updates:

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in any of these up-to-date verified environments with necessary dependencies like [CUDA](https://developer.nvidia.com/cuda), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/):

- **Notebooks** with GPUs: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

The Ultralytics CI tests all [YOLO Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) every 24 hours and at every commit on macOS, Windows, and Ubuntu.

This is an automated response, and an Ultralytics engineer will assist you shortly. Please stay tuned! 😊

**Comment by Kayzwer** [comment]
@GKYoffice yes, if I not mistaken it is for checking AMP and dont worry, your weight will always be load.

**Comment by GKYoffice** [comment]
![3](https://github.com/user-attachments/assets/2b59f7b8-afac-48b1-9d18-e7c47a63ad4f)


Thank you very much!

**Comment by GKYoffice** [comment]
> @GKYoffice是的，如果我没记错的话，这是为了检查AMP，不用担心，你的体重将始终是负担。

Thank you very much
![Uploading 3.png…]()
